src/main/resources/META-INF/plugin.xml

```diff
@@ -85,6 +88,9 @@
                            key="notification.remoteRecording.displayName"/>
         <statusBarWidgetFactory implementation="appland.remote.RemoteRecordingStatusBarFactory"/>
 
+        <notificationGroup id="appmap.telemetry" displayType="BALLOON"
```

Comment:
if you change this to `<notificationGroup id="appmap.telemetry" displayType="STICKY_BALLOON"` (i.e. "STICKY_BALLOON"), then the notification won't expire automatically after 10 second.
I think that's better for important things like telemetry.